client/src/app/navbar/navbar-app/navbar-app.component.ts

```diff
@@ -19,14 +21,33 @@ export class NavbarAppComponent {
   mentees!: UserDTO[] | undefined;
   selectedMenteeName: string = 'Mentorship';
   user$: Observable<UserDTO | null | undefined> = this.fakeUserService.user$;
+  requests!: any[];
+  length!: number;
 
   constructor(
     private userService: UserService,
     private ModalService: NgbModal,
     private menteeService: SharedService,
     private fakeUserService: FakeUserService,
-    private router: Router
-  ) {}
+    private router: Router,
+    private requestService: RequestResponseService
+  ) {
+    // TODO: Decrease it to 1o seconds
+    interval(100000).subscribe((x) => {
```

Comment:
nit: would be create to have something like `const NAVBAR_REQUEST_POLLING_INTERVAL_MILLISECONDS = 100 * 1000` at the top of the file, and then just use it here